.github/workflows/applications.yml

```diff
@@ -0,0 +1,24 @@
+name: Build Pact and run util REPL tests
+
+on:
+  push:
+    branches:
+    - master
```

Comment:
Now that I think about it, I think we can leave these branch restrictions out and have the CI job run on all branches and PRs.  We use them in some other projects when CI is slow, but that's not a problem with this project.

Reply:
Removed!